recipes/cchardet/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cchardet" %}
+{% set version = "1.0.0" %}
+{% set md5 = "353f5ca2bc445cef5f103a9a7c2f7599" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/14/4f/7570c170110a79290824a683f92043ecc048f851e57a2b2e223a9fe5e8c2/cchardet-{{ version }}.tar.gz
```

Comment:
Please use the URL from the [`example`]( https://github.com/conda-forge/staged-recipes/blob/4d2f7fe2712499ec394c8b5a118a8dd91e9ff843/recipes/example/meta.yaml#L18 ).